### ISSUE TITLE
Close tutorial icon appears to screen readers as plain text

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7209]
### Where To Find Bug
Create

### Expected Behaviour
The screen reader should announce the close icon as a button.

### Actual Behaviour
The screen reader announces the close icon as plain text, announcing the name of the icon.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Go to the Create page
- Tab through the page until you get to the tutorial
- Tab to the close icon for the tutorial
- Notice that the screen reader does not identify the icon as a button

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
